src/OutputWriters/jld2_output_writer.jl

```diff
@@ -10,7 +10,10 @@ default_included_properties(::ShallowWaterModel) = [:grid, :coriolis, :closure]
 default_included_properties(::HydrostaticFreeSurfaceModel) = [:grid, :coriolis, :buoyancy, :closure]
 
 update_file_splitting_schedule!(schedule, filepath) = nothing
-update_file_splitting_schedule!(schedule::FileSizeLimit, filepath) = schedule.path = filepath
+
+function update_file_splitting_schedule!(schedule::FileSizeLimit, filepath) 
+    schedule.path = filepath
```

Comment:
```suggestion
    schedule.path = filepath
    return nothing
```

We use explicit returns (if the function is not just one line